### PR TITLE
1093 Update toolstack in README

### PR DESCRIPTION
**Pull request by daveatweaviate**
Add details on installing the toolstack that you need to edit the docs.

Also updates .gitignore to remove the yarn lockfile. 

### What's being changed:

More detailed instructions on what is needed and how to set it up

### Type of change:

- [X] **Documentation** updates (non-breaking change to fix/update documentation) 
  NOTE: This fix is only visible in the repo README.md file, not on the rendered web site

### How Has This Been Tested?

- [X] **Local build** - verified in VS Code preview

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

**Review by databyjp — APPROVED** [review]

**Comment by databyjp** [comment]
Feel free to merge whenever you like @daveatweaviate :)

File: blog/2023-01-03-hybrid-search-explained/index.mdx [hunk 1]
```diff
@@ -18,7 +18,8 @@ The [Hybrid search](/developers/weaviate/api/graphql/search-operators#hybrid) fe
 In this blog post, you will learn about the implementation of hybrid search in Weaviate and how to use it.
 
 ## Sparse and Dense Vectors
-Sparse and dense vectors are calculated with distinct algorithms. Sparse vectors have mostly zero values with only a few non-zero values, while dense vectors mostly contain non-zero values. Sparse embeddings are generated from algorithms like [BM25](https://en.wikipedia.org/wiki/Okapi_BM25) and [SPLADE](https://arxiv.org/abs/2107.05720). Dense embeddings are generated from machine learning models like [GloVe](https://text2vec.org/glove.html) and [Transformers](https://huggingface.co/docs/transformers/index).
+Sparse and dense vectors are calculated with distinct algorithms. Sparse vectors have mostly zero values with only a few non-zero values, while dense vectors mostly contain non-zero values. Sparse embeddings are generated from algorithms like [BM25](https://en.wikipedia.org/wiki/Okapi_BM25) and [SPLADE](https://arxiv.org/abs/2107.05720). Dense embeddings are generated from machine learning models like
```
Comment (daveatweaviate) [inline blog/2023-01-03-hybrid-search-explained/index.mdx:21]: This is an unrelated fix for a build error. The link worked but the line break on a period caused an error
